eng/build.ps1

```diff
@@ -54,8 +55,12 @@ if ($vs) {
     # Enables the logginc of Json RPC messages if diagnostic logging for Test Explorer is enabled in Visual Studio.
     $env:_TestingPlatformDiagnostics_=1;
 
-    # Launch Visual Studio with the locally defined environment variables
-    & "$PSScriptRoot\..\TestFx.sln"
+    if ($vs) {
+        # Launch Visual Studio with the locally defined environment variables
+        & "$PSScriptRoot\..\TestFx.sln"
+    } else {
+        & code "$PSScriptRoot\.."
```

Comment:
You way want to check if `code` exists, in case people (like me) have only `code-insiders`.

Reply:
Thanks @Evangelink and @nohwnd. I'm now checking installation for both code and code-insiders.